clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -322,7 +322,7 @@ class DockerRegistryClient {
         String userDefinedCatalog = new File(catalogFile).getText()
         return (DockerRegistryCatalog) new Gson().fromJson(userDefinedCatalog, DockerRegistryCatalog.class)
       } catch (Exception e) {
-        throw new DockerRegistryOperationException("Unable to read catalog file $catalogFile:" + e.getMessage())
+        throw new DockerRegistryOperationException("Unable to read catalog file $catalogFile: " + e.getMessage())
```

Comment:
since you're changing this
```suggestion
        throw new DockerRegistryOperationException("Unable to read catalog file $catalogFile: " + e.getMessage(), e)
```

Reply:
Fixed in a2a414c